pages/badges/badge-single.jsx

```diff
@@ -234,16 +234,26 @@ var BadgePage = React.createClass({
     );
   },
 
+  getShareCodes: function() {
+    var url = encodeURIComponent(window.location.toString());
+    var msg = encodeURIComponent(`I earned the Mozilla ${this.state.badge.title} badge`);
```

Comment:
@anmechung - any thoughts on copy? This is for the "Share" copy on Badges, where people will be able to post on Twitter and Facebook. Right now we have "I earned the Mozilla [badge name] badge." I'd suggest something like, "I just earned the Mozilla [badge name] Badge! Learn more about Mozilla Badges [link to /credentials]."